sdk/servicebus/microsoft-azure-servicebus/src/main/java/com/microsoft/azure/servicebus/primitives/ConnectionStringBuilder.java

```diff
@@ -95,6 +95,7 @@
      * Connection string value used for the Authentication field which indicates that 
      * Managed Identity TokenProvider will be used for authentication purposes.
      */
+    public static final String MANAGEDIDENTITY_AUTHENTICATION = "ManagedIdentity";
     public static final String MANAGED_IDENTITY_AUTHENTICATION = "Managed Identity";
```

Comment:
I would use MANAGED_IDENTITY_AUTHENTICATION_WITH_SPACE and MANAGED_IDENTITY_AUTHENTICATION_WITHOUT_SPACE as constant names.